homeassistant/components/zwave_mqtt/const.py

```diff
@@ -0,0 +1,190 @@
+"""Constants for the zwave_mqtt integration."""
+DOMAIN = "zwave_mqtt"
+DATA_UNSUBSCRIBE = "unsubscribe"
+PLATFORMS = ["switch"]
+
+# MQTT Topics
+TOPIC_OPENZWAVE = "OpenZWave"
+
+# Common Attributes
+ATTR_INSTANCE_ID = "instance_id"
+ATTR_SECURE = "secure"
+ATTR_CONFIG_PARAMETER = "parameter"
+ATTR_CONFIG_VALUE = "value"
+ATTR_CONFIG_SIZE = "size"
+ATTR_NODE_ID = "node_id"
+ATTR_SCENE_ID = "scene_id"
+ATTR_SCENE_LABEL = "scene_label"
+ATTR_SCENE_VALUE_ID = "scene_value_id"
+ATTR_SCENE_VALUE_LABEL = "scene_value_label"
+
+# Service specific
+SERVICE_ADD_NODE = "add_node"
+SERVICE_REMOVE_NODE = "remove_node"
+SERVICE_REMOVE_FAILED_NODE = "remove_failed_node"
+SERVICE_REPLACE_FAILED_NODE = "replace_failed_node"
+SERVICE_CANCEL_COMMAND = "cancel_command"
+SERVICE_SET_CONFIG_PARAMETER = "set_config_parameter"
+
+# Home Assistant Events
+EVENT_SCENE_ACTIVATED = f"{DOMAIN}.scene_activated"
+
+# Signals
+SIGNAL_DELETE_ENTITY = f"{DOMAIN}_delete_entity"
+
+# Discovery Information
+GENERIC_TYPE_WHATEVER = None  # Match ALL
+SPECIFIC_TYPE_WHATEVER = None  # Match ALL
+SPECIFIC_TYPE_NOT_USED = 0  # Available in all Generic types
+
+GENERIC_TYPE_AV_CONTROL_POINT = 3
+SPECIFIC_TYPE_DOORBELL = 18
+SPECIFIC_TYPE_SATELLITE_RECEIVER = 4
+SPECIFIC_TYPE_SATELLITE_RECEIVER_V2 = 17
+
+GENERIC_TYPE_DISPLAY = 4
+SPECIFIC_TYPE_SIMPLE_DISPLAY = 1
+
+GENERIC_TYPE_ENTRY_CONTROL = 64
+SPECIFIC_TYPE_DOOR_LOCK = 1
+SPECIFIC_TYPE_ADVANCED_DOOR_LOCK = 2
+SPECIFIC_TYPE_SECURE_KEYPAD_DOOR_LOCK = 3
+SPECIFIC_TYPE_SECURE_KEYPAD_DOOR_LOCK_DEADBOLT = 4
+SPECIFIC_TYPE_SECURE_DOOR = 5
+SPECIFIC_TYPE_SECURE_GATE = 6
+SPECIFIC_TYPE_SECURE_BARRIER_ADDON = 7
+SPECIFIC_TYPE_SECURE_BARRIER_OPEN_ONLY = 8
+SPECIFIC_TYPE_SECURE_BARRIER_CLOSE_ONLY = 9
+SPECIFIC_TYPE_SECURE_LOCKBOX = 10
+SPECIFIC_TYPE_SECURE_KEYPAD = 11
```

Comment:
What's these numbers? And should they live in the integration or in the Python OZW MQTT package ?

Reply:
They are used for discovery. I would prefer to move them to OZW MQTT package. @marcelveldt does it make sense to move them?

Reply:
they're being moved to the lib as enums

Reply:
These constants are all moved to the library now

See
https://github.com/cgarwood/homeassistant-zwave_mqtt/pull/119/files
for implementation details

Reply:
Commit done.